tests/cpp/test_gpu1.cpp

```diff
@@ -6753,6 +6753,8 @@ TEST_F(NVFuserTest, FusionPersistentSoftmaxLocalShared_CUDA) {
 
   TensorView* sx_softmax = div(sx_exp, bcast_sum); // (M, N)
   TensorView* dx_softmax = div(dx_exp, bcast_sum); // (M, N)
+  sx_softmax->setContiguity(false);
+  dx_softmax->setContiguity(false);
```

Comment:
wondering what went wrong, Are we generating an output that's not contiguous?

Reply:
This test feeds preallocated outputs that are non-contiguous: https://github.com/NVIDIA/Fuser/blob/61ffac91e715b0f57fd658fb815e2f79a14685cd/tests/cpp/test_gpu1.cpp#L6833-L6834